ios/Scripts/generate_app_icons_from_source.py

```
@@ -0,0 +1,130 @@
+    success_count = sum(
+        resize_icon(source_path, icons_dir, filename, size) for filename, size in icon_sizes.items()
+    )
+
+    print(f"\n🎯 Готово! Создано {success_count}/{total_count} иконок")
+    return success_count == total_count
+
+
+def main() -> None:
+    # Check Pillow availability
+    try:
+        import PIL
```

Flake8 (7.2.0)

[error] 'PIL' imported but unused

(F401)